src/main/python/run_solr_regression.py

```diff
@@ -0,0 +1,204 @@
+# -*- coding: utf-8 -*-
+'''
+Anserini: A Lucene toolkit for replicable information retrieval research
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+'''
+
+import argparse
+import logging
+import math
+import os
+import requests
+import time
+
+from subprocess import call, Popen, PIPE
+
+logger = logging.getLogger('run_solr_regression')
+ch = logging.StreamHandler()
+ch.setFormatter(logging.Formatter('%(asctime)s %(levelname)s - %(message)s'))
+logger.addHandler(ch)
+logger.setLevel(logging.INFO)
+
+class SolrClient:
+    def __init__(self):
+        pass
+
+    def run_shell_command(self, command, echo=False, capture=False):
```

Comment:
Can you abstract this method into a module, something like `regression_utils`? That both the Solr and ES tests use?

Reply:
That would be a good idea! Will do.